bravado/fido_client.py

```diff
@@ -97,6 +97,21 @@ class FidoClient(HttpClient):
     """Fido (Asynchronous) HTTP client implementation.
     """
 
+    def __init__(
+        self,
+        future_adapter_class=None,  # type: typing.Optional[typing.Type[FidoFutureAdapter]]
+        response_adapter_class=None,  # type: typing.Optional[FidoResponseAdapter]
```

Comment:
This should be `typing.Optional[typing.Type[FidoResponseAdapter]]`.

Reply:
Oh, you're right, I'll fix it.